acceptance/bundle/deploy/python-notebook/my_project/databricks.yml.tmpl

```diff
@@ -0,0 +1,18 @@
+bundle:
+  name: my_project
+
+resources:
+  jobs:
+    my_job:
+      name: My Job
+      tasks:
+      - task_key: my_notebook
+        notebook_task:
+          notebook_path: my_notebook.py
+
+targets:
+  dev:
+    mode: development
+    default: true
+    workspace:
+      host: $DATABRICKS_HOST
```

Comment:
Why do we need this here? isn't that the default?

Reply:
Do you mean that databricks.yml doesn't have to declare targets if env variables are set? Note that we don't use "bundle init" here.

Reply:
I mean that $DATABRICK_HOST is already what's going to be used (that's how all other tests work), so no need to specify it explicitly and use .tmpl.

That line can just be deleted, it should work.